captura/views.py

```diff
@@ -257,6 +257,44 @@ def create_estudio(request):
 
 @login_required
 @user_passes_test(is_capturista)
+def estudio_delete_modal(request, id_estudio):
+    """ View to send the form to delete users.
+
+    When a user accesses this view, it returns the form required to
+    confirm the soft delition of a study, along with all of its
+    information.
+
+    Returns
+    ----------
+    GET:
+        On succes returns HTTP 200 with estudio_socioeconomicos/estudio_delete_modal.html
+        template rendeered, this template contains a confirmation form for the soft
+        delition of a estudio socioeconomico
+
+        On error returns HTTP 404
+    """
+    if request.is_ajax():
+        estudio = get_object_or_404(Estudio, pk=id_estudio)
+        form = DeleteEstudioForm(initial={'id_estudio': estudio.pk})
+        return render(request, 'estudios_socioeconomicos/estudio_delete_modal.html',
+                      {'estudio_to_delete': estudio, 'delete_form': form})
+
```

Comment:
you should raise 404 error if you get to the url w/o ajax. it's currently returning None.